cluster/src/test/java/org/apache/iotdb/cluster/log/snapshot/FileSnapshotTest.java

```diff
@@ -301,6 +301,6 @@ public void testInstallPartial()
     for (int i = 0; i < 9; i++) {
       assertEquals(i, loadedFiles.get(i).getMaxPlanIndex());
     }
-    assertEquals(0, processor.getUnSequenceFileList().size());
+    assertEquals(1, processor.getUnSequenceFileList().size());
```

Comment:
Because the logic of judging whether the file exists according to the plan index has changed, the file is not considered to exist when it is equal to. So in the test `testInstallPartial`,  there will be one file to be judged as an unseq file.